seeding/user.seeding.js

```diff
@@ -0,0 +1,83 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import faker from 'faker';
+import UserFactory from '../test/factories/user.factory';
+import VendorFactory from '../test/factories/vendor.factory';
+import AddressFactory from '../test/factories/address.factory';
+import User from '../server/models/user.model';
+import { USER_ROLE } from '../server/helpers/constants';
+import { logLoading, logError, logTable } from './seed-helpers';
+import { ROLE, DEFAULT_PASSWORD } from './seed-constants';
+import { hashPassword } from '../server/services/user.service';
+
+export const seedUsers = async (limit, role, customValues = {}) => {
+  const roleValue = Object.keys(ROLE).find((key) => ROLE[key] === role);
+
+  const hashedPassword = await hashPassword(customValues.password || DEFAULT_PASSWORD);
+
+  const vendor = VendorFactory.build({
+    companyName: faker.company.companyName(),
+    companyLogo: faker.image.abstract(),
+    verified: true,
+    phone: faker.phone.phoneNumber(),
+    bankInfo: {
+      accountName: faker.finance.accountName(),
+      accountNumber: faker.finance.routingNumber(),
+      bankName: faker.company.companyName(),
+    },
+    directors: [
+      {
+        name: faker.name.findName(),
+        isSignatory: true,
+        signature: faker.image.abstract(),
+        phone: faker.phone.phoneNumber(),
+      },
+    ],
+  });
+
+  const users = UserFactory.buildList(limit, {
+    role: roleValue,
+    activated: true,
+    activationDate: new Date(),
+    firstName: faker.name.firstName(),
+    lastName: faker.name.lastName(),
+    email: faker.internet.email(),
+    phone: faker.phone.phoneNumber(),
+    referralCode: `${faker.lorem.word()}${faker.lorem.word()}`,
+    profileImage: faker.image.avatar(),
+    address: AddressFactory.build({
+      city: faker.address.city(),
+      country: faker.address.country(),
+      state: faker.address.state(),
+      street1: faker.address.streetName(),
+      street2: faker.address.streetName(),
+    }),
+    vendor: parseInt(roleValue, 10) === USER_ROLE.VENDOR ? vendor : {},
+    ...customValues,
+    password: hashedPassword,
+  });
+
+  try {
+    if (!roleValue) {
+      logError(`Invalid role input. Available roles are: ${Object.values(ROLE)}`);
+    } else {
+      await User.insertMany(users);
+      logLoading(`${limit} ${parseInt(limit, 10) === 1 ? role : `${role}s`} created`);
+
+      const table = [];
+      users.forEach((user) => {
+        table.push({ email: user.email, password: customValues.password || DEFAULT_PASSWORD });
```

Comment:
Take note of the capital `E` in email and `P` in password.

```suggestion
        table.push({ Email: user.email, Password: password });
```